### PR TITLE
added Lieberherr list

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
#428. Just added [the Lexibank mappings](https://github.com/lexibank/lieberherrkhobwa/blob/master/etc/concepts.csv).

### CODE REVIEW

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -240,3 +240,4 @@ Carling-2017-87k	Carling, Gerd and Larsson, Filip and Cronhamn, Sandra and Farre
...
+Lieberherr-2017-100	Lieberherr, Ismael and Bodt, Timotheus Adrianus	2017		100	basic, areal	English	Kho-Bwa languages	https://escholarship.org/uc/item/4t27h5fg	Lieberherr2017		The given list was compiled to test the relatedness of 22 Kho-Bwa languages, supposedly constituting a separate branch within the Tibeto-Burman subgroup. The analysis of their shared core vocabulary also included 7 other Tibeto-Burman varieties to contrast them with the Kho-Bwa languages. This list is mainly based on the Leipzig-Jakarta 100 concept list (Haspelmath and Tadmor 2009), with some entries, considered ambiguous by the compilers, being replaced with items from [the Swadesh list (1971)](:ref:Swadesh-1971-100).
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv]: this is not an areal list, as it does not occur to be in opposition to normal swadesh.

File: concepticondata/conceptlists/Lieberherr-2017-100.tsv [hunk 2]
```diff
@@ -0,0 +1,101 @@
+NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
```
Comment (LinguList) [inline concepticondata/conceptlists/Lieberherr-2017-100.tsv]: please put ID in first column, we never swap them in this way.

**Comment by chrzyki** [comment]
> nice apart from my two small points. @tresoldi and @chryzki, can you please merge, if you seem fit? I say this as you'll want to mention this in the revision for the MIT chapter.

Will do! I'll wait for @Kristina-Pianykh to fix the small issues and then merge & take note of the revision.

File: concepticondata/conceptlists.tsv [hunk 3]
```diff
@@ -240,3 +240,4 @@ Carling-2017-87k	Carling, Gerd and Larsson, Filip and Cronhamn, Sandra and Farre
...
+Lieberherr-2017-100	Lieberherr, Ismael and Bodt, Timotheus Adrianus	2017		100	basic	English	Kho-Bwa languages	https://escholarship.org/uc/item/4t27h5fg	Lieberherr2017		The given list was compiled to test the relatedness of 22 Kho-Bwa languages, supposedly constituting a separate branch within the Tibeto-Burman subgroup. The analysis of their shared core vocabulary also included 7 other Tibeto-Burman varieties to contrast them with the Kho-Bwa languages. This list is mainly based on the Leipzig-Jakarta 100 concept list (Haspelmath and Tadmor 2009), with some entries, considered ambiguous by the compilers, being replaced with items from [the Swadesh list (1971)](:ref:Swadesh-1971-100).
```
Comment (chrzyki) [inline concepticondata/conceptlists.tsv]: Add proper citation for Haspelmath 2009.

**Pull request merged**